Source/JavaScriptCore/runtime/VM.cpp

```diff
@@ -1558,6 +1559,7 @@ template<typename Visitor>
 void VM::visitAggregateImpl(Visitor& visitor)
 {
     m_microtaskQueue.visitAggregate(visitor);
+    numericStrings.visitAggregateImpl(visitor);
```

Comment:
nit: ditto, just call this `visitAggregate`.

Reply:
Changed.